lib/pacemaker/pcmk_acl.c

```diff
@@ -315,12 +326,20 @@ pcmk__acl_evaled_render(xmlDoc *annotated_doc, enum pcmk__acl_render_how how,
     xslt_ctxt = xsltNewTransformContext(xslt, annotated_doc);
     CRM_ASSERT(xslt_ctxt != NULL);
 
-    if (how == pcmk__acl_render_text) {
-        params = params_noansi;
-    } else if (how == pcmk__acl_render_namespace) {
-        params = params_namespace;
-    } else {
-        params = params_useansi;
+    switch (how) {
+        case pcmk__acl_render_namespace:
+            params = params_namespace;
+            break;
+        case pcmk__acl_render_text:
+            params = params_noansi;
+            break;
+        default:
+            /* pcmk__acl_render_color is the only remaining option.
+             * The compiler complains about params possibly uninitialized if we
+             * don't use default here.
+             */
```

Comment:
FYI, some static analysis will also complain if you put all the enumeration values plus a default with error handling, so you can't win. I generally do it how you have here, with a default catching one known case.